app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,55 @@
+class LessonsController < ApplicationController
+  before_action :authenticate_teacher!, only: %i[create]
```

Comment:
this before action should be for all actions

Reply:
it is now for all actions.